types/trigger.d.ts

```diff
@@ -1,21 +1,18 @@
-import { Lang } from './global';
+import { Lang, NotEnLang } from './global';
 
 export type Matches = {
   target?: string;
   [s: string]: string;
 }
 
+export type TranslatedText = Record<Lang, string>;
 
-type TranslatableText = {
-  [s in Lang]: string;
+export type TranslatableText = {
```

Comment:
It is hard for me to tell the difference between what `TranslatedText` and `TranslatableText` are supposed to mean as type names.

I think my idea with `TranslatableText` was meant to be "this is an object that can be translated" (i.e. is translatable), with "translating" being going from object -> string by selecting one of the properties.  (There's a bunch of functions called `translate` that do this sort of thing.)  `TranslatedText` with past tense "translated", sounds like "this has been translated already", and makes me expect that it's a string.

I really don't have good ideas on what to call these.  `TranslatableText` is already an awkward mouthful.  I'd be hesitant to suggest `FullyTranslatableText` as something to call the `Record<Lang, string>` as that seems even worse.

cc @JLGarber for naming bikesheds if you have any opinions

Reply:
What about `LocaleText`?

Reply:
No, unfortunately I don't think there are any really good options here. I definitely like `LocaleText` in general, or maybe `LocalizableText`? In this case, given how specific we're being with type names, I think we can afford to go a little longer than we normally would for the sake of precision.

Reply:
`LocaleText` is good for me.  What about `FullLocaleText` for the `Record` version?

Reply:
Done